recipes/themysto/meta.yaml

```diff
@@ -0,0 +1,83 @@
+# this is a conda-forge recipe. See https://github.com/conda-forge for details
+
+{% set version = "0.0.4" %}
+
+package:
+  name: themysto
+  version: {{ version }}
+
+source:
+  # also taken from nb_conda
+  git_url: https://github.com/jcb91/IPython-notebook-extensions.git
+  git_tag: conda
+
+build:
+  number: 0
+  # we can't use noarch_python because it places prohibitive restrictions:
+  #  - can't specify supported version
+  #  - attempts to roll its own pre/post-link scripts, complaining about existing ones.
+  entry_points:
+    # Don't put entry points here, as conda bungles the script creation.
+    # Instead rely on setuptools.
+    # The syntax is module:function
+    # - themysto = themysto.application:main
+
+requirements:
+  build:
+    - python 2.7*|3.3*|3.4*|3.5*
```

Comment:
Please remove `python` versions. We already do this for you. Also, we don't build for 3.3.